subprojects/logging/src/main/java/org/gradle/internal/logging/DefaultLoggingConfiguration.java

```diff
@@ -26,7 +26,8 @@
 import java.io.Serializable;
 
 public class DefaultLoggingConfiguration implements Serializable, LoggingConfiguration {
-    private LogLevel logLevel = LogLevel.WARN;
+    public final static LogLevel DEFAULT_LOG_LEVEL = LogLevel.WARN;
+    private LogLevel logLevel;
```

Comment:
:o: I feel conflicted about this change. Without initialization the log level would be null. I am wondering if we could optimize the logic in `LoggingCommandLineConverter.java` on line 84 to catch all cases without having to expose new methods and setting the field to null. Could there be a situation where the end user asks for the log level and it could indeed be null?